installer/README.md

```diff
@@ -1,78 +1,59 @@
-# Cloudflow installer
+# CloudFlow Installer
+This project is the CloudFlow Installer, implemented as a Kubernetes operator.
```

Comment:
This doc should start with a simple step by step how to use a (already distributed somewhere) installer script to install cloudflow, like the old docs.